Source/BlueprintSearchVSExtension/Source/Commands/CommandHelpers/PathFinderHelper.cs

```diff
@@ -29,47 +30,42 @@ public static void FindPaths()
 				ProjectsList.AddRange(ProjectsList[i].ProjectItems.Cast<ProjectItem>().Select(x => x.SubProject).OfType<Project>());
 			}
 
-			foreach (Project project in ProjectsList)
+			if (ProjectsList.Count > 0)
 			{
-				if (project.Name.Equals("UE4"))
+				foreach (Project project in ProjectsList)
 				{
-					// This solution contains a ue4 project, find the path!
-					string EngineIncludePath = GetUnrealCommandLineExecutablePathFromProject(project);
-					if (string.IsNullOrEmpty(EngineIncludePath))
-					{
-						MessageBox.Show("BlueprintSearchVS could not find an unreal project in this solution.", "BlueprintSearchVS Warning");
-					}
-					else
+					if (project.Name.Equals("UE4"))
 					{
 						//We know that we are in a ue4 project, so we analyze all the ue4 metadata we need.
-						UECommandLineFilePath = EngineIncludePath;
 						const string UProjectExtension = ".uproject";
 						UProjectFilePath = Path.GetFullPath(Path.ChangeExtension(DTEService.Solution.FullName, UProjectExtension)).Replace('\\', '/');
+						break;
 					}
+				}
 
+				if (!File.Exists(UProjectFilePath))
+				{
+					MessageBox.Show("BlueprintSearchVS could not find an Unreal project in this solution.", "BlueprintSearchVS Warning");
 					return;
 				}
+
+				string UERootPath = GetUnrealCommandLineExecutablePath();
+				UECommandLineFilePath = Path.Combine(UERootPath, "Engine\\Binaries\\Win64\\UE4Editor-Cmd.exe").Replace('\\', '/');
+				if (!File.Exists(UECommandLineFilePath))
+				{
+					MessageBox.Show("BlueprintSearchVS could not find Unreal's command line executable.", "BlueprintSearchVS Warning");
```

Comment:
```suggestion
					MessageBox.Show("BlueprintSearchVS could not find Unreal's command line executable.\nCheck that you have an up to date Development Editor build.", "BlueprintSearchVS Warning");
```

Providing this makes sense, and newlines work in this context!